scripts/partition_manager_output.py

```diff
@@ -100,19 +117,19 @@ def parse_args():
         description='''Creates files based on Partition Manager results.''',
         formatter_class=argparse.RawDescriptionHelpFormatter)
 
-    parser.add_argument("--input-partitions", required=True, type=str,
-                        help="Path to the input .yml file with partition configuration.")
+    parser.add_argument("--input-partitions", required=True, type=str, nargs="+",
+                        help="Path to the input .yml files, one per domain.")
 
-    parser.add_argument("--input-regions", required=True, type=str,
+    parser.add_argument("--input-regions", required=True, type=str, nargs="+",
                         help="Path to the input .yml file with region configuration.")
 
-    parser.add_argument("--config-file", required=True, type=str,
+    parser.add_argument("--config-file", required=False, type=str,
```

Comment:
The output is generated in two rounds.

First the .config (specified by this argument) is generated per domain.

Lastly, the root image will generate the pm_config.h files, which contain the information collected from all the previously generated .config files.

In the latter case, no .config file is generated, and the `--config-file` argument is omitted.

Reply:
Would it be simpler to generate everything with one call, and pass all the config file paths in a single call?

Reply:
IMO no. This because the "flow" of `partition_manager.cmake` makes the current solution the easiest one to read.

Its like:
```
if not dynamic_image:
    exit

create_local_config

if not root_image:
    export_results
else:
    create_global_config
```

If we want to have a single execution of pm_output for the root image I think the code will become less readable as these checks must be done beforehand, and then set the arguments to the script accordingly.